bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBinding.java

```diff
@@ -108,7 +108,7 @@ public void removeKNXTypeMapper(KNXTypeMapper typeMapper) {
     @Override
     protected void internalReceiveCommand(String itemName, Command command) {
         logger.trace("Received command (item='{}', command='{}')", itemName, command.toString());
-        if (!isEcho(itemName, command)) {
+        if (!KNXConnection.getCompatibilityOH2() && !isEcho(itemName, command)) {
```

Comment:
@kaikreuzer, @watou, Made further Tests. With AutoUpdateBindingProvider KNX1.8 seams to work well in OH1 too.

In OH2 the KNX1.8 bundle now works pretty good and as expected! (without "compatibilityOH2")

I am still doing detailed tests in a OH1 live system. Results will follow.